### PR TITLE
(#57) Implemented Container.stop()

### PR DESCRIPTION
**Pull request by llorllale**
This PR:
* solves #57 
* implements `Container.stop`
* Uses the [Docker API Docs](https://docs.docker.com/engine/api/v1.35/#tag/Container) as reference
* Leaves a  puzzle for the rest

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#73](https://github.com/amihaiemil/docker-java-api/issues/73) is now in scope, role is `REV`

File: src/main/java/com/amihaiemil/docker/Container.java [hunk 1]
```diff
@@ -57,4 +57,10 @@
      * @return String id.
      */
     String containerId();
+
+    /**
+     * Stops the container.
+     * @throws IOException If something goes wrong.
```
Comment (amihaiemil) [inline src/main/java/com/amihaiemil/docker/Container.java]: @llorllale I think we should start specifying:

``
@throws UnexpectedResponseException If the status response is not <<expected>>
``
as well, not just ``IOException``.
Don't you think?

Comment (llorllale) [reply src/main/java/com/amihaiemil/docker/Container.java]: @amihaiemil done

**Comment by llorllale** [comment]
@amihaiemil done, please review

**Comment by amihaiemil** [comment]
@rultor merge it

**Comment by amihaiemil** [comment]
@rultor merge again?

**Pull request merged**